### PR TITLE
Homework 2. Arrays and cycles

### PR DESCRIPTION
**Pull request by slchk**

### CODE REVIEW

File: task-2-3/Program.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                int maxNumber = array[0];
+                for (int i = 1; i < array.Length; i++)
+                {
+                    if (array[i] > maxNumber)
+                    {
+                        maxNumber = array[i];
+                    }
+                }
+
+                Console.WriteLine($"Maximum value in array is {maxNumber}");
```
Comment (VladimirVolkovVD) [inline task-2-3/Program.cs]: array.Max()

File: task-2-3/Program.cs [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                int minNumber = array[0];
+                for (int i = 1; i < array.Length; i++)
+                {
+                    if (array[i] < minNumber)
+                    {
+                        minNumber = array[i];
+                    }
+                }
+
+                Console.WriteLine($"Minimum value in array is {minNumber}");
```
Comment (VladimirVolkovVD) [inline task-2-3/Program.cs]: array.Min();

File: task-2-4/Program.cs [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] numbers = { 1, 2, 3, 5 };
+            Console.WriteLine(string.Join(", ", numbers));
+            int sum = 0;
+
+            foreach (int i in numbers)
```
Comment (VladimirVolkovVD) [inline task-2-4/Program.cs]: foreach (var number in numbers)

File: task-2-4/Program.cs [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+            {
+                sum += i;
+            }
+
+            double average = (double)sum / numbers.Length;
+            
+            int[] numbers2 = { 9, 15, 63 };
+            Console.WriteLine(string.Join(", ", numbers2));
+            int sum2 = 0;
+            foreach (int j in numbers2)
```
Comment (VladimirVolkovVD) [inline task-2-4/Program.cs]: foreach (var number in numbers2)

File: task-2-4/Program.cs [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+            int sum = 0;
+
+            foreach (int i in numbers)
+            {
+                sum += i;
+            }
+
+            double average = (double)sum / numbers.Length;
+            
+            int[] numbers2 = { 9, 15, 63 };
```
Comment (VladimirVolkovVD) [inline task-2-4/Program.cs]: secondNumberArray

File: task-2-4/Program.cs [hunk 6]
```diff
@@ -0,0 +1,47 @@
+﻿using System.Diagnostics.CodeAnalysis;
+
+namespace MyApp
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            int[] numbers = { 1, 2, 3, 5 };
```
Comment (VladimirVolkovVD) [inline task-2-4/Program.cs]: firstNumberArray

File: task-2-2/Program.cs [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+            Console.WriteLine("Enter the number");
+            string userInput = Console.ReadLine();
+           
+            if (int.TryParse(userInput, out int number))
+            {
+                if (array.Contains(number))
+                {
+                    int newArrayIndex = 0;
+                    int newArrayLength = 0;
+                    foreach (int i in array)
```
Comment (VladimirVolkovVD) [inline task-2-2/Program.cs]: i j k mostly for for cycle, in foreach you go object by object in collection and you can provide more informative names foreach (int number in array)

**Review by VladimirVolkovD — APPROVED** [review]
Small issue

File: task-2-4/Program.cs (hunk 3 above)
Comment (slchk) [reply task-2-4/Program.cs]: could you explain please what for?

**Review by VladimirVolkovTDS — APPROVED** [review]

**Pull request merged**